doc/classes/@GDScript.xml

```diff
@@ -778,14 +801,14 @@
 			<argument index="1" name="step" type="float">
 			</argument>
 			<description>
-				Snap float value to a given step.
+				Snaps float value "s" to a given "step".
```

Comment:
Use [code] tag?

Reply:
Like:
Snaps float value [code]"s"[/code] to a given [code]"step"[/code]
?

Reply:
About the previous example, it seems obvious now that the quotes should not be in that.

Reply:
Yes, [code]s[/code] turns into `s`.